### ISSUE TITLE
applicable to multichannel mask (multiclass) predictions ? meaning each class is trained on different channel

### ISSUE THREAD

**Issue by amardeepjaiman** [Vooban/Smoothly-Blend-Image-Patches#7]
Can this script be used to innference multiclass on multiple channels ? We have multichannel masks prepared while training . Please suggest.

**Comment by bnsreenu** [comment]
Yes, this can be used for multichannel predictions, just define nb_classes = 6 (in my case I have 6 classes).

**Comment by bhaswara** [comment]
Hi @bnsreenu, I would like to ask question about this. I trained my model to accept two inputs with different channel. How to use this script to accept two inputs? I always get an error
